apps/data-types.md

```diff
@@ -75,6 +75,16 @@ This object is used for the attributes of shippingAddress and billingAddress.
 | subCategories | array of [link](page:apps-data-types#link) | A list of links to the subcategories. |
 | sfUrl | string | The link to the categories in the shop’s storefront. |
 
+## category (create request)
+
+| Attribute | Type | Description |
+| - | :-: |  - |
+| name | string | The name of the category. |
+| alias | string | The alternate name that uniquely identifies the category. If already in use, the system will automatically increment by number, e.g. alias1, alias2.  |
```

Comment:
The unique identifier of the new category. If already in use, the system will automatically increment by number, e.g. alias1, alias2.